scimma_admin/scimma_admin/settings.py

```diff
@@ -118,24 +119,22 @@ def get_rds_db(db_instance_id):
     }
 
 
-# Password validation
-# https://docs.djangoproject.com/en/3.0/ref/settings/#auth-password-validators
+# Authentication
+# https://mozilla-django-oidc.readthedocs.io/en/stable/settings.html
 
-AUTH_PASSWORD_VALIDATORS = [
-    {
-        'NAME': 'django.contrib.auth.password_validation.UserAttributeSimilarityValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.MinimumLengthValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.CommonPasswordValidator',
-    },
-    {
-        'NAME': 'django.contrib.auth.password_validation.NumericPasswordValidator',
-    },
-]
+AUTHENTICATION_BACKENDS = (
+    'mozilla_django_oidc.auth.OIDCAuthenticationBackend',
+)
+OIDC_RP_CLIENT_ID = "cilogon:/client_id/29ad6d4e4198bd207e916128fec3c656"
```

Comment:
So, according to the docs, this one should also come from secrets--maybe put this in AWS secrets as well.

https://mozilla-django-oidc.readthedocs.io/en/stable/installation.html#add-settings-to-settings-py

Reply:
Oh, interesting. That wasn't my impression from reading OAuth docs like https://www.oauth.com/oauth2-servers/client-registration/client-id-secret/ - I thought the client ID is just a public identifier.

In our case, if an attacker has the client ID, all they can do is craft a request to CILogon claiming to be the scimma-admin application. But CILogon will check that the redirect URI is `https://admin.dev.hop.scimma.org/auth/callback/` and reject the request if it's anything else. If it _is_ that, then the net result is that the attacker just got a user's browser to open the SCiMMA admin page, which doesn't seem bad?

Anyway, I'll ask Warren. The OIDC application might need to be destroyed and remade in order to rotate this credential, so I can't change it myself.

Reply:
Yeah, I believe your description of the auth flow is correct. It's likely an extra layer of security--far be it from me to decry what may be a best practice. Warren will probably know best.

Reply:
Confirmed that it's not a secret, but we rotated credentials anyway during the uncertainty and I've pushed a change which pulls the value from secrets manager.